src/container/srv_target.c

```diff
@@ -443,17 +410,15 @@ cont_aggregate_ult(void *arg)
 				DP_RC(rc));
 			/* Sleep 2 seconds when last aggregation failed */
 			msecs = 2ULL * 1000;
-		} else if (rc == 2) {
-			/* Sleep 2 seconds when VOS aggregation skipped */
-			msecs = 2ULL * 1000;
 		} else {
-			msecs = 1ULL * 200;
+			if (msecs == 0)
+				msecs = 2ULL * 100;
```

Comment:
I don't get this, why do we need to sleep 200 ms?

Reply:
The original implementation will sleep 200ms in this case. So I just follow it.